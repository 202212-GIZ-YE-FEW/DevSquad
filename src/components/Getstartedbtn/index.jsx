import Link from "next/link";
const Getstartedbtn = () => {
    return (
        <div>
            <Link
                href='/blogs'
                className='w-20 h-7 py-2.5 px-4 rounded-lg bg-primary-blue text-white text-2xl font-medium'
            >
                Get Started
            </Link>
        </div>
    );
};

export default Getstartedbtn;
