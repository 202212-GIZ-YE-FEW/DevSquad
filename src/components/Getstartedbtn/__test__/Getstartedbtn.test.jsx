import renderer from "react-test-renderer";
import Getstartedbtn from "../Index";

it("renders correctly", () => {
    const tree = renderer.create(<Getstartedbtn />).toJSON();
    expect(tree).toMatchSnapshot();
});
