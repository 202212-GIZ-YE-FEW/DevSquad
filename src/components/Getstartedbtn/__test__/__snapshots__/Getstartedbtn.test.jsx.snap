// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <a
    className="w-80 h-7 py-2.5 px-4 rounded-lg overflow-hidden group bg-primary-blue relative hover:bg-gradient-to-r hover:from-primary-blue hover:to-blue-200 text-white text-xl font-medium hover:ring-2 hover:ring-offset-2 hover:ring-primary-blue transition-all ease-out duration-300"
    href="/signup"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onTouchStart={[Function]}
  >
    <span
      class="absolute right-0 w-8 h-32 -mt-12 transition-all duration-1000 transform translate-x-12 bg-white opacity-10 rotate-12 group-hover:-translate-x-40 ease"
    />
    <span
      class="relative"
    >
      home.GetStarted
    </span>
  </a>
</div>
`;
